applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/child_image/multiprotocol_rpmsg/prj.conf

```diff
@@ -0,0 +1,39 @@
+#
```

Comment:
Isn't there a way to use the default configuration + overlay instead of copy-pasting it for each application?

Reply:
The "default configuration" means the configuration from a sample. I was always wondering why we depend on it as these configurations were never hardened.

At the moment I don't think it will work but I can check what would need to be changed. The new scheme has the same steps for config files merging as zephyr.

Reply:
Copy-pasting its configuration won't remove the dependency on the sample, so I guess we must accept the fact that the sample is not used as a sample, but as a foundational component :). Either way, I think we should always try to reduce the amount of configuration in our applications and samples so that users who want to base their applications on ours don't need to update their configuration with each NCS release (of course, such a need may still arise, but it can be avoided in some scenarios, why not?).

Reply:
> Copy-pasting its configuration won't remove the dependency on the sample

I will remove the dependency on its configuration, which is good enough.

> the sample is not used as a sample, but as a foundational component

This is something that was brought up few months ago already but will not be fixed for 1.8.

> I think we should always try to reduce the amount of configuration in our applications and samples so that users who want to base their applications on ours don't need to update their configuration with each NCS release (of course, such a need may still arise, but it can be avoided in some scenarios, why not?).

I agree, this is why having one configuration file is better then two. Having a clearly defined place where configuration is stored is better then tracking from whatever file in the system configuration is taken.
But I agree that creating a specific configuration file may not be welcomed as it requires taking responsibility whereas if taking the so called default configuration moves it to... nobody I guess (sample is just a sample).

Anyway, it is your app. I will check what can be done to have sample config merged too.

Reply:
We can leave move the location but keep the file names as before. Then configuration files are overlayed.
I have added a small comment in each to mention what is the original source of configuration.

Reply:
Thanks!